net/kadnode/Makefile

```diff
@@ -0,0 +1,92 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=kadnode
+PKG_VERSION:=2.0.2
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/mwarning/KadNode.git
+PKG_SOURCE_VERSION:=26dbc9436f8d2c68b899cd32e01bdbe1565294b2
+
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_MIRROR_HASH:=67ba2ce03c3dd74b547885891391657aa9d6fae7c14558634776e1111e7cbe57
+PKG_LICENSE:=MIT
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/kadnode
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=P2P DNS Server
+	SUBMENU:=IP Addresses and Names
+	URL:=https://github.com/mwarning/KadNode
+	MENU:=1
+	DEPENDS:=+KADNODE_ENABLE_BOB:libmbedtls +KADNODE_ENABLE_TLS:libmbedtls +KADNODE_ENABLE_UPNP:libminiupnpc +KADNODE_ENABLE_NATPMP:libnatpmp
+	MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+endef
+
+define Package/kadnode/description
+	P2P DNS resolver
```

Comment:
I'd _love_ to see more information here personally.  THis is what's available to people in menuconfig, and it would be nice to have _something_ descriptive here.  THink of it as a teaser

Reply:
What about "Decentralized DNS system based on the BitTorrent network"? I am bit unimaginative here.. It can be used as a decentralized DynDNS replacement.